Makefile

```diff
@@ -14,6 +14,19 @@ conformance-test: ## Run the conformance test suite
 proto: ## Rebuild protobuf autogenerated code
 	protoc --go_out=pkg api/sbom.proto
 
+.PHONY: proto-gorm
+proto-gorm: gorm.proto proto ## Compile protobufs with protoc-gen-gorm plugin
+	protoc --gorm_out=pkg api/sbom.proto
+
+PROTOC_GORM_VER := d3024d4fa7c9ce9d62e2f2cc0b42faf679211846
+PROTOC_GORM_URL := https://raw.githubusercontent.com/infobloxopen/protoc-gen-gorm/${PROTOC_GORM_VER}/proto/options/gorm.proto
+
+.SILENT: gorm.proto
+gorm.proto:
+	echo Downloading protoc-gen-gorm...
+	go install github.com/infobloxopen/protoc-gen-gorm@$(PROTOC_GORM_VER)
```

Comment:
Will this override an existing protoc-gen-gorm install? It might be good to make this an explicit command rather than an implicit global side effect.

Reply:
It installs it into `$GOPATH/pkg/mod`, but doesn't update go.mod for the project. Multiple versions of protoc-gen-gorm can be installed simultaneously so it won't be overridden

Reply:
While it's not obligatory for this pull request, I propose we move it to .tmp (refer to my patch in an alternative format) with a prefix:```
GOBIN=$(shell realpath $(TEMPDIR)) go install ..
```
This approach directs all build tools to the designated temporary folder instead of relying on the local GOPATH. Although it might not make a difference in the development container, I believe it's worth considering.

@puerco, what are your thoughts on installing all our tools under .tmp within the local repository?

Reply:
@manifestori